bear.py

```diff
@@ -0,0 +1,176 @@
+import os
+import sys
+import argparse
+import constants
+import random
+
+from field import out_file
+
+
+def generate_fighting_coef():
+    return random.random()
+
+
+def end_of_fight(field, winner, looser, winners_fighting_coef, loosers_fighting_coef):
+    winner.life_count += loosers_fighting_coef
+    looser.life_count -= winners_fighting_coef
+
+    if looser.life_count <= 0:
+        looser.die(field)
+    else:
+        looser.go_to_random_cell(field)
+
+
+class Bear(object):
+
+    def __init__(self, name, activity_level, smell, aggression, cowardice, life_count, cell_row, cell_col):
+
+        self.life_count = life_count
+        self.last_action = constants.REST
+        self.last_direction = constants.MOVEMENT_RANDOM
+        self.activity_level = activity_level
+        self.smell = smell
+        self.aggression = aggression
+        self.cowardice = cowardice
+        self.cell_row = cell_row
+        self.cell_col = cell_col
+        self.name = name
+
+    def print_bear_data(self):
+        print >> out_file, 'B{} life count {} last action {} location r {} c {}'.format(self.name, self.life_count, self.last_action, self.cell_row, self.cell_col)
+
+    def eat_honey(self, field):
+        cell = field.get_cell_at_location(self.cell_row, self.cell_col)
+
+        if not cell.has_honey:
+            return
+
+        do_eat = random.random()
+
+        if do_eat <= self.smell:
+            print >> out_file, 'Bear {} ate honey'.format(self.name)
+            self.life_count += 1
+            self.activity_level /= 3
```

Comment:
I think they meant the bear should have a lower activity level for the next turn, not all of them
He's going to enter a coma if you keep dividing it

Reply:
you are right, fixed